portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,25 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&family=Nunito&display=swap" rel="stylesheet">
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div id="header">
+            <div id="left"><img src="/images/profile.jpg" id="profile"></div>
+            <div id="right">
+                <h1>Hi, I'm Keely!</h1>
+                <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and 
+                    business). Outisde of school, I'm an avid hiker, recreational tennis player, and huge bookworm. 
+                    Currently re-reading <i>Gone with the Wind</i> - any book recommendations are always welcome! 
```

Comment:
My favorite book series is the Kingkiller chronicle. I recommend trying 'The name of the wind' if you haven't already.

Reply:
Cool, I'll definitely check that out!